src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -46,6 +48,9 @@ public Subsystems() {
 		} else {
 			launcherSubsystem = null;
 		}
+		if (LIMELIGHT_ENABLED) {
+			limelightSubsystem = new LimelightSubsystem();
+		}
```

Comment:
```suggestion
		} else {
		    limelightSubsystem = null;
		}
```
(For consistency with the other subsystems)

Reply:
Done